src/page-modules/assistant/trip/trip-pattern/trip-pattern.module.css

```diff
@@ -140,3 +140,14 @@
   gap: var(--spacings-small);
   align-items: center;
 }
+
+.timeStartContainer {
+  display: flex;
+  direction: row;
+  gap: 4px;
```

Comment:
Should use `var(--spacings-xSmall)` instead. Should be the same size.

Reply:
Will fix👍